tests/data/raml_10/data-type-examples.raml

```diff
@@ -0,0 +1,150 @@
+#%RAML 1.0
+
+title: Example
+baseUri: https://example.test/{param}/
+
+baseUriParameters:
+  param:
+    type: string
+    description: account name
+    example: splat
+
+mediaType: application/json
+protocols: [ HTTPS ]
+
+types:
+
+  with_multiple_examples:
+    type: string
+    examples:
+      simple: "abc"
+      fancy: "two words"
+      excessive:
+        displayName: "Serious Overkill"
+        description: |
+          There's way too much text here.
+        value:
+          error: This should not be a map.
```

Comment:
The specific example value should not have been a map, because the parameter type is specified as ``string``.  I'll adjust the example for clarity.